paper-typeahead-input.html

```diff
@@ -449,6 +450,14 @@
         // }
       }
     },
+    _candidatesUpdated:function(candidates){
+      if(this.inputValue != undefined){
+        var val = this.inputValue.trim();
+        if(candidates.length > 0 && candidates[0].trim() !== val){
```

Comment:
Correct and it will do the search because the inputValue will be some version of the word "red", the first element in the old array is "red" and the new first element is "white".